rancher-cluster-operations/bulk-components/scripts/destroy_components.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash -e
+IFS=$'\n'
+
+terraform workspace select default
+var_file="${1:-terraform.tfvars}"
+workspace_prefix="${2:-bulk}"
+workspaces=$(terraform workspace list | grep "${workspace_prefix}" | sed 's/*/ /' | sort -r )
+timeout=${3:-30}
+
+for workspace in ${workspaces}; do
+	workspace="$(echo -e ${workspace} | tr -d '[:space:]')"
+	if [ ${workspace} == "default" ]; then
+		continue
+	fi
+  echo "destroying workspace: ${workspace}"
+	terraform workspace select "${workspace}"
+	terraform destroy --auto-approve -var-file=${var_file} "${@:4}"
```

Comment:
I'm not sure when Terraform started to allow "--" but do you think we should keep this consistent?

Reply:
In my experience it has no effect when '-' is followed by another '-', but I have removed it for consistency.